webservices/rest.py

```diff
@@ -219,6 +219,16 @@ def add_aggregate_resource(api, view, schedule, label):
     '/committee/<committee_id>/filings/',
     '/candidate/<candidate_id>/filings/',
 )
+
+api.add_resource(
+    filings.EFilingSummaryView,
+    '/efile/presidential/<string:form_type>/',
```

Comment:
Yea, my former self did not have as good of ideas about urls, ;) happy to hammer it out among our future selves- they will have even better ideas!